pyrsa/inference/noise_ceiling.py

```diff
@@ -42,10 +43,10 @@ def cv_noise_ceiling(rdms, ceil_set, test_set, method='cosine',
         train = ceil_set[i]
         test = test_set[i]
         pred_train = pool_rdm(train[0], method=method)
-        pred_train = pred_train.subsample_pattern(by=pattern_descriptor, 
+        pred_train = pred_train.subsample_pattern(by=pattern_descriptor,
                                                   value=test[1])
```

Comment:
not completely sure but should this perhaps be `train[1] ` instead of ` test[1]` ?

Reply:
I didn't change anything about this here, but test[1] are the patterns sampled for the test set. Thus in this line we choose from pred_train to the patterns in the test set. That seems sensible to me.